docs/security/cosmosdb-jupyter-notebook-2021-08-12.md

```diff
@@ -0,0 +1,66 @@
+
+# Content
+This document describes the mitigation of a security issue described [here](https://msrc-blog.microsoft.com/2021/08/27/update-on-vulnerability-in-the-azure-cosmos-db-jupyter-notebook-feature/).
+
+# Prerequisite
+The Industrial-IoT Platform has been deployed as described in the [Industrial-IoT github repository](https://github.com/Azure/Industrial-IoT/blob/main/docs/deploy/readme.md).
+The steps below refers to a new full installation of the Industrial-IoT Platform.
+
+# Validate proper operation
+- Open the "Engineering Tool" of the Industrial-IoT Platform of the deployment.
+- Goto "Discovery" and "Turn on" "Scanning" on linuxgateway, if it is not already on.
+On a new deployment this should result in:
+[09/05/2021 16:11:01] linuxgateway0-lbwvpab_module_discovery: Completed.
+[09/05/2021 16:11:00] linuxgateway0-lbwvpab_module_discovery: 3/3: ... 3 servers found - 2 endpoints found on opc.tcp://linuxgateway0-lbwvpab-sim0.internal.cloudapp.net:51201/...
+
+- Goto "Browse" and "Turn on" "Activation Action" on "opc.tcp://10.1.8.5:51200/
```

Comment:
... when deployed simulation, otherwise use the your own available OPC UA server

Reply:
addressed in the prerequisites section now.